src/utils/daos_hdlr.c

```diff
@@ -2244,83 +2064,45 @@ file_lstat(struct cmd_args_s *ap, struct file_dfs *file_dfs,
 		/* POSIX returns -1 on error and sets errno
 		 * to the error code
 		 */
-		if (rc != 0) {
+		if (rc != 0)
 			rc = errno;
-		}
 	} else if (file_dfs->type == DAOS) {
-		rc = stat_dfs(ap, file_dfs, path, buf);
+		rc = dfs_sys_stat(file_dfs->dfs_sys, path, 0, buf);
```

Comment:
Technically lstat should be:
```suggestion
		rc = dfs_sys_stat(file_dfs->dfs_sys, path, O_NOFOLLOW, buf);
```
Otherwise links are followed by default